server/index.js

```diff
@@ -1,10 +1,13 @@
-// define server here
+var path = require('path');
 var express = require('express');
 var bodyParser = require('body-parser');
```

Comment:
You're importing the "body-parser" module but not using it anywhere in the code below.